ddtrace/contrib/asgi/middleware.py

```diff
@@ -83,7 +83,7 @@ def span_from_scope(scope):
 
 
 def _request_blocked(span):
-    return span and config._appsec_enabled and _context.get_item(WAF_CONTEXT_NAMES.BLOCKED, span=span)
+    return span and config._appsec_enabled and core.get_item(WAF_CONTEXT_NAMES.BLOCKED, span=span)
```

Comment:
Not quite part of this PRs change but related question: will we always include trying to pull the value off of the span in `core.get_item` or is that just being added as a temporary route and will eventually be deprecated and removed when we fully replace the span context with ExecutionContext value?

Reply:
Yes, I'm treating the storage of contextual data on spans as a stopgap measure that enables incremental change and will be removed in the future.